src/omotes_simulator_core/solver/network/assets/production_asset.py

```diff
@@ -149,7 +149,7 @@ def get_pre_scribe_mass_flow_or_pressure_equations(
                 ]
             )
             equation_object.coefficients = np.array([-1.0 + 2 * connection_point])
-            equation_object.rhs = self.mass_flow_rate_set_point
+            equation_object.rhs = -self.mass_flow_rate_set_point
```

Comment:
Does this follow the correct definition? 

If connection_point = 0, coefficent = +1 and rhs = -m_dot. So mass flow from node to asset is then negative?

Reply:
if conpoint = 0 then coef is -1, and rhs - m_dot. So flow from node to asset is positive. So indeed this is wrong. error is in the heat demand cluster in which we approach as a producer, but we do not set the heat demand to be negative thermal power. I adjusted this.